2-mandatory/3-financial-times.js

```diff
@@ -23,15 +31,26 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  articlesWithNumbers = [];
```

Comment:
What is missing from this line of code? Why does it still work without the missing piece of code?